lib/format-number-string.ts

```diff
@@ -5,7 +5,7 @@ export default (num: number, {
   formatThousands = false,
 } = {}): string => {
   /* string→number coercion after toFixed is for removing trailing zeros when they're existed: 17.00 → 17 */
-  const numStr = (+num.toFixed(2)).toString();
+  const numStr = (+num).toFixed(2);
```

Comment:
Not sure that I've got the idea behind this change.

The comment above was saying what was going on here. The comment hasn't been changed, so I assume that there was an error, but not the line is fully “in accordance” with the comment.

Then, I'm testing the change:

```js
const num = 17.0001;

console.log((+num.toFixed(2)).toString()); // '17'

console.log((+num).toFixed(2)); // '17.00'
```

The current implementation does not remove trailing zeros, while the comment says it should.

If the idea was to change the actual behavior, then it's better to add a flag or something, because there are users who rely on the current behavior. So it should not be changed, I believe.

Reply:
@leusrox, ah, I see.

I'd rather believe types, than README.

I think that this part in README was simply copy-pasted from `formatPhoneNumberString` description. But we are free to change the helpers to make them allow strings:

#41

Reply:
Thanks!